### PR TITLE
chore: Re-format Haskell code

### PR DESCRIPTION
**Pull request by eriksvedang**

### CODE REVIEW

File: src/Concretize.hs [hunk 1]
```diff
@@ -525,9 +526,9 @@ instantiateGenericStructType typeEnv originalStructTy@(StructTy _ _) genericStru
               nameFixedMembers = renameGenericTypeSymbolsOnProduct renamedOrig memberXObjs
               validMembers = replaceGenericTypeSymbolsOnMembers mapp nameFixedMembers
               concretelyTypedMembers = replaceGenericTypeSymbolsOnMembers mappings memberXObjs
-           -- We only used the renamed types for validation--passing the
-           -- renamed xobjs further down leads to syntactical issues.
-           in case validateMembers typeEnv renamedOrig validMembers of
+           in -- We only used the renamed types for validation--passing the
+              -- renamed xobjs further down leads to syntactical issues.
+              case validateMembers typeEnv renamedOrig validMembers of
```
Comment (hellerve) [inline src/Concretize.hs:531]: This is a weird reformat for me

File: src/Eval.hs [hunk 2]
```diff
@@ -89,30 +89,33 @@ eval ctx xobj@(XObj o info ty) preference resolver =
         unwrapLookup =
           fromMaybe
             (throwErr (SymbolNotFound spath) ctx info)
-        -- | Try all lookups performs lookups for symbols based on a given
-        -- lookup preference.
+
```
Comment (hellerve) [inline src/Eval.hs:92]: Where did this comment go?

Comment (hellerve) [reply src/Eval.hs:92]: It might be because it’s a docstring comment with `|`?

File: src/Concretize.hs (hunk 1 above)
Comment (eriksvedang) [reply src/Concretize.hs:531]: It's what Ormolu did...

File: src/Eval.hs (hunk 2 above)
Comment (eriksvedang) [reply src/Eval.hs:92]: Huh, weird. Will add it back – thanks for noticing!

File: src/Concretize.hs (hunk 1 above)
Comment (eriksvedang) [reply src/Concretize.hs:531]: I can't find a good way around it, unfortunately 🤷‍♂️

**Comment by scolsen** [comment]
Thanks!

**Pull request merged**